### PR TITLE
New package: Schuischta.SystemLoadTracker version 1.1

### PR DESCRIPTION
**Pull request by Schuischta**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/140706)

### CODE REVIEW

**Comment by Schuischta** [comment]
> @Schuischta please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

File: manifests/s/Schuischta/SystemLoadTracker/1.1/Schuischta.SystemLoadTracker.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Schuischta.SystemLoadTracker
+PackageVersion: "1.1"
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: SystemLoadTracker.exe
+  PortableCommandAlias: portable
+Installers:
```
Comment (stephengillie) [inline manifests/s/Schuischta/SystemLoadTracker/1.1/Schuischta.SystemLoadTracker.installer.yaml:11]:
```suggestion
Dependencies:
  PackageDependencies:
   - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
Installers:
```

(Automated response - build 808.)

Comment (Schuischta) [reply manifests/s/Schuischta/SystemLoadTracker/1.1/Schuischta.SystemLoadTracker.installer.yaml:11]: Hi @stephengillie,

do i have to change something? If so what exactly do i have to change?

Thanks!

**Review by stephengillie — APPROVED** [review]

**Pull request merged**